src/intl/en/learn-quizzes.json

```
@@ -326,6 +326,51 @@
+  "i002-d-label": "All of the above",
+  "i002-d-explanation": "A DAO can have a multitude of 'missions'.",
+  "i003-prompt": "Unlike traditional organizations, DAOs are…",
+  "i003-a-label": "Usually hierarchical",
+  "i003-a-explanation": "DAOs are usually flat, and fully democratized.",
+  "i003-b-label": "Transparent and fully public about their activities",
+  "i003-b-explanation": "Thanks to on-chain voting, decisions are transparent on the blockchain. Discussions and other elements of the decision-making process are open to all members.",
+  "i003-c-label": "Controlled by a central party",
+  "i003-c-explanation": "Changes require voting by the members. Services offered are handled automatically in a decentralized manner.",
+  "i003-d-label": "Restricted regarding who can suggest changes",
+  "i003-d-explanation": "Usually, every DAO member can suggest changes.",
+  "i004-prompt": "What is essential about smart contracts for DAOs?",
+  "i004-a-label": "The smart contract code can be modified",
+  "i004-a-explanation": "Once the contract is live on Ethereum, no one can change the rules except by a vote. This allows the DAO to run by the rules it was programmed with.",
+  "i004-b-label": "It has an individual owner who retains authority to make changes and send from the treasury.",
+  "i004-b-explanation": "The treasury is defined by the smart contract. To spend money, the group's approval is needed.",
+  "i004-c-label": "Trust in the underlying blockchain due to a distributed consensus",
+  "i004-c-explanation": "It’s important for a DAO that the underlying blockchain cannot be manipulated. Ethereum’s own consensus is distributed and established enough for organizations to trust the network.",
+  "i004-d-label": "DAOs don’t need Smart Contracts",
+  "i004-d-explanation": "The backbone of a DAO is its smart contract, which defines the rules of the organization and holds the group's treasury.",
+  "i005-prompt": "What is not a mechanism to govern a DAO?",
+  "i005-a-label": "Token-based membership",
+  "i005-a-explanation": "Token-based governance is very widely used. It’s usually fully permissionless and is typically used to govern broad decentralized protocols and/or tokens themselves.",
+  "i005-b-label": "Share-based membership",
+  "i005-b-explanation": "Share-based DAOs are more permissioned but still quite open. Any prospective member can submit a proposal to join the DAO, usually offering a tribute of some value in the form of tokens or work.",
+  "i005-c-label": "Reputation-based membership",
+  "i005-c-explanation": "Unlike token or share-based membership, reputation-based DAOs don't transfer ownership to contributors. DAO members must earn reputation through participation.",
+  "i005-d-label": "Residency-based membership",
+  "i005-d-explanation": "There are a lot of possible governance mechanisms in DAOs, but residency is not a criterion. On the contrary, DAOs are characterized by their global nature.",
```

@JanetMo What about a mechanism that relies entirely on off-chain mechanisms, such as "Executive board and off-chain treasury management"

Maybe something like this?

```suggestion
  "i005-d-label": "Executive board and off-chain treasury management",
  "i005-d-explanation": "This approach uses highly centralized and opaque mechanisms of governing. On the contrary, DAOs use verifiable voting mechanisms and on-chain treasury management to ensure transparency and accountability."
```

---

Gonna go ahead with this.. @JanetMo please still feel free to raise any suggested changes!